Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include "SourceAssetsStorage.h"
+#include "RobotImporter/Utils/RobotImporterUtils.h"
```

Comment:
This include can omit "RobotImporter/Utils"
```suggestion
#include "RobotImporterUtils.h"
```